core/src/main/java/bisq/core/offer/placeoffer/bisq_v1/tasks/CheckNumberOfUnconfirmedTransactions.java

```diff
@@ -14,7 +14,7 @@ public CheckNumberOfUnconfirmedTransactions(TaskRunner<PlaceOfferModel> taskHand
     @Override
     protected void run() {
         if (model.getWalletService().isUnconfirmedTransactionsLimitHit() || model.getBsqWalletService().isUnconfirmedTransactionsLimitHit())
-            failed(Res.get("shared.unconfirmedTransactionsLimitReached"));
+            return failed(Res.get("shared.unconfirmedTransactionsLimitReached"));
```

Comment:
The curly brackets at the if are missing. The failed() method has void as return type so it cannot be used as return statement.